File: 1480. Running Sum of 1d Array/answer.swift
// https://leetcode.com/problems/running-sum-of-1d-array/
class Solution {
  func runningSum(_ nums: [Int]) -> [Int] {
      var ret: [Int] = []
      for e in nums {
          ret.append(e + (ret.last ?? 0))
      }
      return ret
  }
}

File: 1470. Shuffle the Array/answer.js
// https://leetcode.com/problems/shuffle-the-array/
/**
 * @param {number[]} nums
 * @param {number} n
 * @return {number[]}
 */
var shuffle = function(nums, n) {
    let ret = [];
    for (let i = 0; i < n; ++i) {
        ret.push(nums[i])
        ret.push(nums[n + i])
    }
    return ret;
};
